sdks/java/maven-archetypes/java8-examples/src/main/resources/archetype-resources/src/main/java/DebuggingWordCount.java

```diff
@@ -0,0 +1,164 @@
+/*
```

Comment:
Yes, it makes sense to not check these in. I don't know off the top of my head the best way to do that. Filed [BEAM-1005](https://issues.apache.org/jira/browse/BEAM-1005).